.github/workflows/ci.yml

```diff
@@ -854,6 +854,13 @@ jobs:
           image_url: ghcr.io/getsentry/relay:${{ github.event.pull_request.head.sha || github.sha }}
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
+      - name: Inspect failure
+        if: failure()
```

Comment:
TIL that there is a `failure()`

Reply:
Love the [docs](https://docs.github.com/en/actions/reference/evaluate-expressions-in-workflows-and-actions#failure) on this:

> Returns true when any previous step of a job fails. If you have a chain of dependent jobs, failure() returns true if any ancestor job fails.